ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -337,6 +338,14 @@ private void createBanner() {
             vungleBannerAd = mVungleManager.getVungleBanner(mPlacementForPlay, mAdConfig.getAdSize(), mVunglePlayListener);
             if (vungleBannerAd != null) {
                 updateVisibility();
+                RelativeLayout.LayoutParams adViewParams = (RelativeLayout.LayoutParams) vungleBannerAd.getLayoutParams();

```

Comment:
For smart/adaptive banners, `adLayout` width maybe larger than `vungleBannerAd` so we need let subview getting centered. Detailed comments see: https://vungle.atlassian.net/browse/AND-2664